browser/js/friend/friend.html

```diff
@@ -4,6 +4,7 @@
   <md-button class="md-raised md-primary" ng-click="follow(friend.id)" ng-show="userFriendsIds.indexOf(friend.id) === -1">Follow {{friend.name}}</md-button>
   <md-button class="md-raised" ng-show ="userFriendsIds.indexOf(friend.id) !== -1" ng-click="unfollow(friend.id)">Unfollow {{friend.name}}</md-button>
   <h4>Learning:</h4>
+  <md-button class="md-raised md-primary" ng-diabled=true ng-show ="userFriendsIds.indexOf(friend.id) !== -1">You Follow {{friend.name}}</md-button>
```

Comment:
The way Ingrid and I had the button on friend profile page was "FOLLOW {{friend.name}}" (class="md-raised md-primary") and "UNFOLLOW {{friend.name}}" (class="md-raised"). Clicking the button would toggle the button between the two states. Maybe change the button & add the functionality here so that it's consistent?